### ISSUE TITLE
Style the Thank You/Email Signup page for Thunderbird

### ISSUE THREAD

**Issue by edrushka** [mozilla/donate.mozilla.org#1357]
Style this page for the Thunderbird flow (this will be used for both the Standard and With Context donate pages).

I suggest:
1) Add the blue background
2) Replace the heart with the TB logo
3) Remove the connected globe graphic
4) Change the header copy to "From the Thunderbird team, Thank You "
5) Change the body copy to "Sign up to receive email updates on Thunderbird."
6) Change privacy notice to "I'm OK with Thunderbird handling...." and link to the TB-specific policy here https://www.mozilla.org/en-US/privacy/thunderbird/

Anything else @rkent?
![8m75etqq9zx4xcl_a3duij9mqrl7qsb9tdsnomev8fc](https://cloud.githubusercontent.com/assets/4515662/13435859/3fe608ce-df91-11e5-96ff-25e00c910bdc.png)

**Comment by rkent** [comment]
1) The main text should be changed, "open web" is Firefox-speak. To keep it simple, how about "We're working hard to protect your inbox." 

2) "the non-profit organization behind Thunderbird" not Firefox.

**Comment by edrushka** [comment]
@rkent - how's this for copy:

Header: From the Thunderbird team, Thank You
Body: We're working hard to protect your inbox. Sign up to receive email updates on Thunderbird. 
Privacy notice: I'm OK with Thunderbird handling my info as explained in this privacy policy. [link to the TB-specific policy here https://www.mozilla.org/en-US/privacy/thunderbird/]

As for the footer, if we change the footer text it will need to be universal across the entire flow. Flagging @davidascher to confirm we can alter the last line of the footer text to read "Mozilla is the non-profit organization behind Thunderbird."

**Comment by edrushka** [comment]
Also, rkent: confirming that you're ok with donors from *all* locales hitting this page after they donate.

**Comment by rkent** [comment]
On locales: How do you usually handle localization? Will any of the flow that we have been talking about be localized?

I'm also not really satisfied with "Sign up to receive email updates on Thunderbird". More so than with Firefox, the intent of the list will be to get non-binding guidance from our donors about the direction of Thunderbird, including how donations are used.

How about this change: "Sign up to receive email messages concerning Thunderbird, including updates and requests for feedback."

**Comment by edrushka** [comment]
Perfect. That copy is great.

Re: localization - Yes, this flow will be localized. @alicoding can provide more detail.
Re: locales - The fundraising team only sends donors from EN and DE locales to the email signup. With Exact Target, we can segment these groups and send them English and German emails. Donors from all other locales go straight to the social share page.

**Comment by vazquez** [comment]
#### Update

- Design updates have been made to make this a TB page
- Copy has been changed as per @rkent's suggestion

TODO:

- Decision from @rkent as to whether or not we should show the /thank-you page to _all locales_.

![screen shot 2016-03-02 at 10 18 11](https://cloud.githubusercontent.com/assets/1119821/13464722/7dabb8be-e060-11e5-99bc-31fc243efb0c.png)

**Comment by vazquez** [comment]
Footer copy has been changed. This change appears only in the Thunderbird donation flow.

![screenshot 2016-03-02 14 47 27](https://cloud.githubusercontent.com/assets/1119821/13473143/fdc4ef64-e085-11e5-9d18-003c03e9acad.png)

**Comment by rkent** [comment]
@vazquez : The body text does not yet include all of the suggested changes. In particular, "We want to keep you in the loop!" should be changed to "Sign up to receive email messages concerning Thunderbird, including updates and requests for feedback.".

Does this page then use the Mail Chimp API to directly attempt a signup, or does it need to go through a Mail Chimp template page first?

**Comment by rkent** [comment]
@edrushka Yes I want all locales to go through this page, not just EN and DE.

**Comment by vazquez** [comment]
Sorry @rkent, forgot to update that screenshot.

![screen shot 2016-03-02 at 15 04 20](https://cloud.githubusercontent.com/assets/1119821/13473630/2af84d76-e088-11e5-9c49-9154a5a58a5e.png)

**Comment by vazquez** [comment]
We're not using MailChimp for collection of emails. We use ExactTarget on the donate platform.

**Comment by vazquez** [comment]
Confirmed. Users from _all locales_ will now go through this page.

**Comment by rkent** [comment]
@vazquez Yes I understand that Mozilla uses ExactTarget, but we've had discussions that it would be better for Thunderbird to use an alternate platform, and we've been looking at MailChimp.

For Exact Target, I assume that adding the email address here will result in their name being added to an email list, with I assume some tagging about the original source of their contact. We would want to do the same thing with Mail Chimp.
